assets/js/scenes/welcomeScene.js

```diff
@@ -64,7 +65,7 @@ const createWelcomeScene = () => {
     addButton("Start", vec2(80, 350), () => {
       play("menu_select", { loop: false, volume: 1.0 });
       introMusic.stop();
-      go("game", { score: 0, livesLeft: 3 });
+      go("game", { score: 0, livesLeft: 3,isIntroMusicPaused: isIntroMusicPaused,});
```

Comment:
Suggestion (Non-blocking): You may not need the is `IntroMusicPaused: isIntroMusicPaused`. You should be able to use just `isIntroMusicPaused`.